<!--REQUEST_ID=d388eb16-eba8-44e9-bf58-781b193d6711-->
Failure on x86_64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.47922&attempt_id=9dcdce2d-9f4a-42a7-b303-d1847a8d4827)

Attempted: slurm

<details><summary>Partial log (click to expand)</summary><p>

```
collect2: error: ld returned 1 exit status
make[3]: *** [Makefile:578: sbcast] Error 1
make[3]: Leaving directory '/build/source/src/sbcast'
make[2]: *** [Makefile:563: all-recursive] Error 1
make[2]: Leaving directory '/build/source/src'
make[1]: *** [Makefile:690: all-recursive] Error 1
make[1]: Leaving directory '/build/source'
make: *** [Makefile:589: all] Error 2
builder for '/nix/store/c4sdajh1sszivganr7dp00w2jdbqz2y6-slurm-18.08.0-1.drv' failed with exit code 2
error: build of '/nix/store/c4sdajh1sszivganr7dp00w2jdbqz2y6-slurm-18.08.0-1.drv' failed
```
</p></details>

--- a/pkgs/servers/computing/slurm/default.nix
+++ b/pkgs/servers/computing/slurm/default.nix
@@ -1,5 +1,5 @@
 { stdenv, fetchFromGitHub, pkgconfig, libtool, curl
-, python, munge, perl, pam, openssl
+, python, munge, perl, pam, openssl, zlib
 , ncurses, mysql, gtk2, lua, hwloc, numactl
 , readline, freeipmi, libssh2, xorg, lz4
 # enable internal X11 support via libssh2
@@ -34,7 +34,7 @@ stdenv.mkDerivation rec {
 
   nativeBuildInputs = [ pkgconfig libtool ];
   buildInputs = [
-    curl python munge perl pam openssl
+    curl python munge perl pam openssl zlib
       mysql.connector-c ncurses gtk2 lz4
       lua hwloc numactl readline freeipmi
   ] ++ stdenv.lib.optionals enableX11 [ libssh2 xorg.xauth ];
@@ -42,9 +42,10 @@ stdenv.mkDerivation rec {
   configureFlags = with stdenv.lib;
     [ "--with-freeipmi=${freeipmi}"
       "--with-hwloc=${hwloc.dev}"
-      "--with-lz4=${lz4}"
+      "--with-lz4=${lz4.dev}"
       "--with-munge=${munge}"
       "--with-ssl=${openssl.dev}"
+      "--with-zlib=${zlib}"
       "--sysconfdir=/etc/slurm"
     ] ++ (optional (gtk2 == null)  "--disable-gtktest")
       ++ (optional enableX11 "--with-libssh2=${libssh2.dev}");

